Submission of Lab 01 by BT23F05F063

Longshlur opened the pull request:
complete lab 1

Longshlur committed:
i have complete my lab 1

s-m-quadri commented:
@bot-s-m-quadri status

bot-s-m-quadri commented:
## 📌 PRN: BT23F05F063

We've successfully received your submission for **Lab 01**.

**Your PRN**: `BT23F05F063`  
**Lab Number**: 01  
**Base Branch**: `lab-daa-01`

### All your submissions so far:
- #88
- #44

Please ensure you complete all required files for this lab.  
We will review your submissions **within at most 3 weeks**, i.e., by **05 September 2025**.

If you have doubts or questions, comment here, and we’ll get back to you  
once you’ve completed your work.

_Thank you for your submission._

Longshlur committed:
Add all files

Longshlur committed:
Completed all Python files:
- countdown.py: Recursive and iterative countdown implementation
- factorial.py: Recursive and iterative factorial calculation
- print_letters.py: Print letters recursively and iteratively
- sum_numbers.py: Sum of first n numbers using both approaches
- task_print_array_backward.py: Array reverse printing implementation
- task_reverse_digits.py: Number digit reversal functions
- task_sum_even_numbers.py: Sum of even numbers implementation

s-m-quadri committed:
Merge branch 'sub-lab-daa-01' into lab-daa-01

s-m-quadri commented:
👍🏻 Good. Seems alright to me.

@bot-s-m-quadri merge it now.

bot-s-m-quadri commented:
👍 Got it, I’ll start merging and organizing your submission.
